### PR TITLE
Fix vivado_alt_synth flow (v2)

### PR DESCRIPTION
**Pull request by makslevental**
Sorry for taking so long with this. I've incorporated your comments from https://github.com/XedaHQ/xeda/pull/37. Only major difference between this PR and that one is I dropped the changes to `sqrt.toml` and my user supplied XDC hacky solution (which you implemented in https://github.com/XedaHQ/xeda/commit/163149e9d87b026afefd394a9644b27cf639bfd7).

### CODE REVIEW

File: src/xeda/flows/flow.py [hunk 1]
```diff
@@ -108,6 +108,7 @@ def _validate_quiet(cls, value, values):
         )
         no_console: bool = Field(False, hidden_from_schema=True)
         reports_dir: str = Field("reports", hidden_from_schema=True)
+        checkpoints_dir: str = Field("checkpoints", hidden_from_schema=True)
```
Comment (makslevental) [inline src/xeda/flows/flow.py:111]: Copy+pasting comment chain from the previous PR here:

[kammoh](https://github.com/kammoh) [14 days ago](https://github.com/XedaHQ/xeda/pull/37#discussion_r962021074)
Is this coming from the stale branch or an intended change?

Author
@[makslevental](https://github.com/makslevental) makslevental [7 minutes ago](https://github.com/XedaHQ/xeda/pull/37#discussion_r976679726)
This is intended - [vivado_alt_synth.tcl has set checkpoints_dir `settings.checkpoints_dir`](https://github.com/XedaHQ/xeda/blob/main/src/xeda/flows/vivado/templates/vivado_alt_synth.tcl#L11) but [flow.py does not set this field](https://github.com/XedaHQ/xeda/blob/main/src/xeda/flows/flow.py#L110).

**Comment by kammoh** [comment]
Thank you for your contribution. It looks great!
Could you just please rebase/merge from main, so hopefully the CI tests will pass?

File: src/xeda/flows/flow.py (hunk 1 above)
Comment (kammoh) [reply src/xeda/flows/flow.py:111]: Yes, my bad!

**Comment by makslevental** [comment]
> Thank you for your contribution. It looks great! Could you just please rebase/merge from main, so hopefully the CI tests will pass?

Rebased. Let's see how the tests go...

**Review by kammoh — APPROVED** [review]

**Comment by kammoh** [comment]
LGTM :ship:
Great work. Thank you!

**Pull request merged**